pdi-dataservice-client/pom.xml

```diff
@@ -85,6 +85,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pdi-dataservice-client-plugin-api</artifactId>
+      <version>${pdi.version}</version>
```

Comment:
This should be `${project.version}` instead of `${pdi.version}`

Reply:
That was changed in the core pom.xml in this commit. It was an eng-ops task, so somehow I assume that it's better this way, regarding versions management.
https://github.com/pentaho/pdi-dataservice-plugin/commit/81080e9c2f96790d2ef86aad4c017377a86103a2

Reply:
is `pdi-dataservice-client-plugin-api` a module of this project, then it should be `project.version`

Reply:
The `pdi-dataservice-client-plugin-api` module didn't even exist when that commit was done :P
That module is part of the maven project being defined therefore it gets the `${project.version}` for version.

Reply:
fixed